src/dump.c

```diff
@@ -1927,7 +1927,7 @@ static int size_isgreater(const void *a, const void *b)
     return *(size_t*)b - *(size_t*)a;
 }
 
-static jl_value_t *read_verify_mod_list(ios_t *s, arraylist_t *dependent_worlds, jl_array_t *mod_list)
+static jl_value_t *read_verify_mod_list(jl_module_t *from, ios_t *s, arraylist_t *dependent_worlds, jl_array_t *mod_list)
```

Comment:
Oops: should this be called `from` or `into`? I'm not sure what this module is for. @JeffBezanson, @vtjnash?

Reply:
I think this is `into` since it should verify the module list with respect to the environment the module is loaded into.

Reply:
This isn't loading a module into an environment, it's just parsing a file and asserting that the header looks valid. @StefanKarpinski You should drop all changes to this file; my earlier PR to loading.jl moved everything into Julia that matters for Pkg3.